MapleServer2/Types/GlobalEvent.cs

```diff
@@ -1,42 +1,60 @@
-﻿using MapleServer2.Packets;
+﻿using MapleServer2.Data.Static;
+using MapleServer2.Packets;
 using MapleServer2.Servers.Game;
 using MapleServer2.Tools;
 
 namespace MapleServer2.Types;
 
 public class GlobalEvent
 {
-    public readonly int Id;
+    public int Id;
+    public int FirstHour;
+    public int FirstMinutesOnHour;
+    public int MinutesToRunPerDay;
     public List<GlobalEventType> Events = new();
 
     public GlobalEvent()
     {
         Id = GuidGenerator.Int();
     }
 
-    public async Task Start()
+    public void Start()
     {
-        MapleServer.BroadcastPacketAll(GlobalPortalPacket.Notice(this));
+        // Assign new ID
+        Id = GuidGenerator.Int();
+        GameServer.GlobalEventManager.AddEvent(this);
+        BroadcastEvent();
```

Comment:
You have asynchronous code here that is not being awaited, which may cause various issues in a multi threaded application like this one. Start ought to be asynchronous.